helm/ffc-mpdp-backend/values.yaml

```diff
@@ -43,3 +43,16 @@ readinessProbe:
   initialDelaySeconds: 10
   periodSeconds: 10
   failureThreshold: 3
+
+postgresService:
+  postgresDb: ffc_mpdp_backend
+  postgresqlHost: ffc-mpdp-backend
+  postgresPort: 5432
+  postgresSchema: public
+  postgresUser: postgres
+
+aadPodIdentity: true
+
+azureIdentity:
+  clientID: not-a-real-clientID
+  resourceID: not-a-real-resourceID
```

Comment:
For Pod identity to work need to pass these values to the Helm chart from Azure App Config.  Needs to be set up